ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,36 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="torch.result_type",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        num_arrays=2,
+    ),
+)
+def test_torch_result_type(
+    dtype_and_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    tensor1_dtype, tensor1 = input_dtype[0] , x[0]
+    tensor2_dtype, tensor2 = input_dtype[1] , x[1]
+    helpers.test_frontend_function(
```

Comment:
Please add `tensor1` and `tensor2` as arguments to this `test_frontend_function`